### DISCUSSION TITLE
[Design] Dataspace Connector Logo

### DISCUSSION THREAD

**Discussion by DominikPinsel** [eclipse-edc/.github#14] [Planning]
I think having a logo helps the project a lot. We can use it in the documentation, so that the reader can recognize the project easier. It also helps the project to look more professional and makes it easier to sell our product to people, who are not involved yet.
Additionally the image can be used in various documents or presentations, which again, helps the project to look more professional.

I think getting a logo should not be that complicated. Maybe some of us know some designers in our companies or other talented employees which can help us out here. What do you think?

<sub>Dominik Pinsel <dominik.pinsel@daimler.com>, Daimler TSS GmbH, [legal info/Impressum](https://github.com/Daimler/daimler-foss/blob/master/LEGAL_IMPRINT.md)</sub>

**Comment by boorich** [comment]
Is this being worked on? Do you need any support @mspiekermann ?

**Comment by mspiekermann** [comment]
We are working on some drafts for the logo. But ofc feel free to contribute you ideas @empea-careercriminal, much appreciated!

**Comment by mspiekermann** [comment]
So far, we have not been able to find a suitable logo that abstractly represents the vision and idea of the EDC. I would be happy if we still receive ideas and suggestions from the community here.
